src/components/Calendar/Calendar.tsx

```diff
@@ -55,7 +57,9 @@ const CalendarStyling = styled.div`
     .fc .fc-icon {
         font-size: min(4vw, 1.2rem);
     }
-
+    .fc .fc-scrollgrid-section-header.fc-scrollgrid-section-sticky > * {
```

Comment:
Can you explain what the > * is doing?

Reply:
that just means that it targets all direct descendants of `.fc-scrollgrid-section-header.fc-scrollgrid-section-sticky`